templates/register.html

```diff
@@ -26,15 +29,13 @@
 {% block content %}
 <div class="container">
 {% if registered %}
-<div class="margins">
-<div class="heading">Thankyou for registering.</div>
-<div class="sub-heading"><a href={% url 'home' %}>Return to the home page</a></div>
+<h2>Thank you for registering.</h2>
```

Comment:
I can't comprehend the size of the text, a screenshot would be great if possible for you.

Reply:
![register](https://user-images.githubusercontent.com/37906201/67488583-ab59cf00-f635-11e9-819a-20e5838e0b46.png)